docs/kubernetes-installers.md

```diff
@@ -39,9 +39,18 @@ want. Once the cluster has been created, this CIDR cannot be changed.
 
 ### Rancher
 
-When creating a workload cluster, set the [network
-plugin](https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins/)
-to `none`.
+Follow these steps to deploy Antrea(as a [custom CNI](https://rke.docs.rancher.com/config-options/add-ons/network-plugins/custom-network-plugin-example)) on [Rancher](https://ranchermanager.docs.rancher.com/pages-for-subheaders/installation-and-upgrade) cluster via UI.
```

Comment:
the link provided for `Rancher` does not seem to be relevant. Shouldn't we use a link like this one: https://ranchermanager.docs.rancher.com/pages-for-subheaders/kubernetes-clusters-in-rancher-setup ?